client/allocrunner/networking_bridge_linux.go

```diff
@@ -195,10 +225,16 @@ const nomadCNIConfigTemplate = `{
 						{
 							"subnet": %q
 						}
+					],
+					[
+						{ 
+							"subnet": %q
```

Comment:
again I'm wondering if there's a way this could introduce problems on systems that don't ipv6.  have you tried disabling ipv6 on a test machine to see whether the bridge plugin errors trying to use this config?

the bridge plugin does not like it, then we'll need to generate this template differently somehow, so it can conditionally add ipv6 subnet/dst only if the system supports it.